static/app/components/events/eventStatisticalDetector/lineChart.tsx

```diff
@@ -32,47 +35,27 @@ function LineChart({statsData, evidenceData, start, end, chartLabel}: ChartProps
   const theme = useTheme();
   const router = useRouter();
 
-  const results = transformEventStats(statsData, chartLabel);
-  const {smoothedResults, minValue, maxValue} = transformEventStatsSmoothed(
```

Comment:
Ah, I didn't realize the transform was applying the min and max values on the chart. I think we can remove it for now because the smoothing is giving you issues, but I think it would be a good idea to reapply this independently.

Not sure if you had opinions but I think it would get rid of some whitespace so we can show the change better

Reply:
Most charts are using the defaults without specifying the min/max for the y axis as it's fairly reasonable. We can reassess later if we really want to constrain the y axis values later.